### PR TITLE
feat: Parse correct upstream from SENTRY_DSN env

### PR DESCRIPTION
**Pull request by sl0thentr0py**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -116,6 +131,7 @@ fn make_config() -> Result<Config> {
     let overrides = OverridableConfig {
         mode: Some("proxy".to_string()),
         shutdown_timeout: Some(SHUTDOWN_TIMEOUT.to_string()),
+        upstream: upstream_url(),
```
Comment (antonpirker) [inline src/main.rs:134]: Does this fall back to "https://sentry.io" in case no SENTRY_DSN env var is defined?

Comment (antonpirker) [reply src/main.rs:134]: maybe trough the `..Default` below?

Comment (sl0thentr0py) [reply src/main.rs:134]: yes, see https://github.com/getsentry/relay/blob/a66c0eb945f66d25e862245d4f40b88a9ae2e77f/relay-config/src/config.rs#L441-L454

Comment (antonpirker) [reply src/main.rs:134]: ok

**Review by antonpirker — APPROVED** [review]

**Pull request merged**